docs/build/js-client/3-hello-world.md

```diff
@@ -63,10 +63,10 @@ Aqua comes with the standard library, which can be accessed from "@fluencelabs/a
 npm install --save-dev @fluencelabs/aqua-lib
```

Comment:
is that needed given that cli is installed?

Reply:
I think it's better to use specific version listed in your package.json instead of latest version on cli. Thinks can break after you update cli and new aqua-lib will change it's interfaces/behavior

Reply:
Yeap, no need. Removed this